test/runtests.jl

```diff
@@ -284,6 +289,12 @@ end
         @test FASTQ.hasquality(record)
         @test FASTQ.quality(record) == b"@BCFFFDFHHHHHJJJIJIJJIJJJJJJJJIJJJJIIIJJJIJJJ" .- 33
 
+        record = FASTQ.Record("id", "desc", "AAGCT", collect("@BCFF"))
```

Comment:
This should be `record1` I think...

Reply:
Whoops. Last minute change on my part to rename the variable